rules_scheduler/rules_scheduler.install

```diff
@@ -115,9 +115,169 @@ function rules_scheduler_update_1201() {
+                },
+                "fields": {
+                    "tid": {
+                        "id": "tid",
+                        "table": "rules_scheduler",
+                        "field": "tid"
+                    },
+                    "config": {
+                        "id": "config",
+                        "table": "rules_scheduler",
+                        "field": "config",
+                        "alter": {
+                            "make_link": true,
+                            "path": "admin/config/workflow/rules/components/manage/[config]"
+                        }
+                    },
+                    "date": {
+                        "id": "date",
+                        "table": "rules_scheduler",
+                        "field": "date"
+                    },
+                    "identifier": {
+                        "id": "identifier",
+                        "table": "rules_scheduler",
+                        "field": "identifier"
+                    },
+                    "nothing": {
+                        "id": "nothing",
+                        "table": "views",
+                        "field": "nothing",
+                        "label": "Operations",
+                        "alter": {
+                            "text": "delete",
+                            "make_link": true,
+                            "path": "admin/config/workflow/rules/schedule/[tid]/delete",
+                            "alt": "Delete this scheduled task"
+                        }
+                    }
+                },
+                "sorts": {
+                    "date": {
+                        "id": "date",
+                        "table": "rules_scheduler",
+                        "field": "date"
+                    }
+                },
+                "arguments": {
+                    "config": {
+                        "id": "config",
+                        "table": "rules_scheduler",
+                        "field": "config",
+                        "default_argument_type": "fixed",
+                        "summary": {
+                            "format": "default_summary"
+                        },
+                        "limit": "0"
+                    }
+                },
+                "filters": {
+                    "config": {
+                        "id": "config",
+                        "table": "rules_scheduler",
+                        "field": "config",
+                        "exposed": true,
+                        "expose": {
+                            "operator_id": "config_op",
+                            "label": "Component filter",
+                            "operator": "config_op",
+                            "identifier": "config",
+                            "remember": true
+                        }
+                    }
+                }
+            }
+        }
+    }
+}';
+    $data = backdrop_json_decode($data);
```

Comment:
Since you took the step of hard-coding this view (which is correct), let's hard-code this as an array instead of as a json string. This way you will avoid using backdrop_json_decode(), which is a function that may or may not be available. I feel that we have try to get things done in ways that are "tried and true" rather than assume things will work. As I pointed out before, we should follow the pattern use in core in places like `user_update_1025()`

Reply:
Updated to use the PHP version of the View.

Incidentally, I wonder if there is an easier way to get the PHP version than what I did. Since Exporting a view now exports it as JSON (versus the PHP version one could get in D7), what I did was to run this snippet in the "Execute PHP Code" Devel page:

```
$view = views_get_view('rules_scheduler');
$text = var_export($view);
dpm($text);
```

Then copy the resulting text and hand-edit it to adhere to B code standards and replace two calls to `::__set_state()`.

Reply:
I've  used [Default Views Convert to Config](https://github.com/backdrop-contrib/default_views_config) and then, once you have the json file, `backdrop_json_decode`.